lib/storage/index_db.go

```diff
@@ -2654,6 +2662,11 @@ func matchTagFilters(mn *MetricName, tfs []*tagFilter, kb *bytesutil.ByteBuffer)
 	return true, nil
 }
 
+func isCompositeFilter(tfss []*TagFilters) bool {
```

Comment:
It's misleading function name or incorrect implementation, `isCompositeFilter` should return bool if there are 2 filters, one of them is metric name selector and second one a key=value filter.  as an example: `name{filter="value"}`.